common/static/coffee/spec/discussion/view/new_post_view_spec.coffee

```diff
@@ -10,21 +10,29 @@ describe "NewPostView", ->
         )
         @discussion = new Discussion([], {pages: 1})
 
-    checkVisibility = (view, expectedVisible, expectedDisabled) =>
-      view.render()
-      expect(view.$(".js-group-select").is(":visible")).toEqual(expectedVisible)
-      disabled = view.$(".js-group-select").prop("disabled")
-      if expectedVisible and ! expectedDisabled
+    checkVisibility = (view, expectedVisible, expectedDisabled, noRender) =>
+      if !noRender
```

Comment:
Alright. This will mean I need to update all the other functions that call it, however. I figured this would have been the lesser evil, but I'll get to work on that. :)